tests/functions.py

```diff
@@ -17,22 +18,25 @@
 
 
 def GET(testpath, **optional):
-    if "api" in optional:
-        api_v = optional.get('api', None)
-        if api_v == "1":
-            api_url = api1_url
-        elif api_v == "2":
-            api_url = api2_url
-        else:
-            raise ValueError('api parameter should be "1" or "2"')
+    if 'https' in testpath or 'http' in testpath:
+        getit = requests.get(testpath)
     else:
-        api_url = default_api_url
-    if optional.pop("anonymous", False):
-        auth = None
-    else:
-        auth = authentification
-    getit = requests.get(api_url + testpath, headers=header,
-                         auth=auth)
+        if "api" in optional:
+            api_v = optional.get('api', None)
+            if api_v == "1":
+                api_url = api1_url
+            elif api_v == "2":
+                api_url = api2_url
+            else:
+                raise ValueError('api parameter should be "1" or "2"')
```

Comment:
"Should" is a recommendation.  If it has to be 1 or 2, use "must".
```suggestion
                raise ValueError('api parameter must be "1" or "2"')
```